### DISCUSSION TITLE
Spatial symmetries

### DISCUSSION THREAD

**Discussion by Shovan-Physics** [block-hczhai/block2-preview#135] [Q&A] [answered]
This is probably a wider question about DMRG, but I was wondering if it is possible to implement spatial symmetries in `block2`. For example, if I have a 1D spin chain that is symmetric with respect to reflection `R` about the center, can I impose `R=1` or `R=-1`?

**Comment by hczhai** [comment, marked as answer]
This can be done using custom symmetry groups. I have added an example in the updated documentation https://block2.readthedocs.io/en/latest/tutorial/custom-hamiltonians.html#Spatial-Reflection-Symmetry.

**Reply by Shovan-Physics** [reply]
Thanks!
